<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paint</class>
 <widget class="QWidget" name="Paint">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>Colour</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Back to board</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
